Cargo.toml

```diff
@@ -24,9 +24,9 @@ froggy = "0.4"
 genmesh = "0.5"
 gfx = "0.16"
 gfx_glyph = "0.3"
-gltf = "0.9.1"
-gltf-importer = "0.9.1"
-gltf-utils = "0.9.1"
+gltf = { features = ["names"], version = "0.9.2" }
+gltf-importer = { features = ["names"], version = "0.9.2" }
```

Comment:
I wonder if `gltf` can follow `serde` an expose external crates in public via features (e.g `features = ["names", "importer", "utils"]`)

Reply:
That's a pretty good idea. I'll make an issue for that.